src/tests/ascent/t_ascent_mpi_relay_extract.cpp

```diff
@@ -966,6 +966,301 @@ TEST(ascent_relay, test_relay_mpi_sparse_topos_2)
 
 }
 
+//-----------------------------------------------------------------------------
+TEST(ascent_relay, silo_spiral_multi_file)
+{
+    //
+    // Set Up MPI
+    //
+    int par_rank;
+    int par_size;
+    MPI_Comm comm = MPI_COMM_WORLD;
+    MPI_Comm_rank(comm, &par_rank);
+    MPI_Comm_size(comm, &par_size);
+
+    CONDUIT_INFO("Rank "
+                  << par_rank
+                  << " of "
+                  << par_size
+                  << " reporting");
+
+    Node n;
+    ascent::about(n);
+
+    //
+    // Create an example mesh.
+    //
+    Node data, verify_info;
+
+    // use spiral , with 7 domains
+    conduit::blueprint::mesh::examples::spiral(7,data);
+    add_matset_to_spiral(data, 7);
+
+    // rank 0 gets first 4 domains, rank 1 gets the rest
+    if(par_rank == 0)
+    {
+        data.remove(4);
+        data.remove(4);
+        data.remove(4);
+    }
+    else if(par_rank == 1)
+    {
+        data.remove(0);
+        data.remove(0);
+        data.remove(0);
+        data.remove(0);
+    }
+    else
+    {
+        // cyrus was wrong about 2 mpi ranks.
```

Comment:
I still love this comment

Reply:
it lives on, across projects, space, and time